client-side/src/jest.config.js

```diff
@@ -0,0 +1,20 @@
+// module.exports = {
+//     testEnvironment: 'jsdom',
+//     setupFilesAfterEnv: ['<rootDir>/setupTests.js'],
+//     moduleNameMapper: {
+//         '\\.(css|scss)$': 'identity-obj-proxy',
+//     },
+// };
+// module.exports = {
+//     transformIgnorePatterns: [
+//       "/node_modules/(?!axios)"
```

Comment:
1. remove comments
2. take pull from main branch and then be sure that `npm test` command is still running as expected after this change